### ISSUE TITLE
No properties file found for [classpath:i18n/messages_en_US] - neither plain properties nor XML

### ISSUE THREAD

**Issue by Cyrus254Kenya** [WebGoat/WebGoat#995]
2021-05-03 12:01:59.265 DEBUG 1843 --- [  XNIO-1 task-2] org.owasp.webgoat.i18n.Messages          : Loading properties [messages.properties] with encoding 'UTF-8'
2021-05-03 12:01:59.270 DEBUG 1843 --- [  XNIO-1 task-2] org.owasp.webgoat.i18n.Messages          : No properties file found for [classpath:i18n/messages_en] - neither plain properties nor XML
2021-05-03 12:01:59.272 DEBUG 1843 --- [  XNIO-1 task-2] org.owasp.webgoat.i18n.Messages          : No properties file found for [classpath:i18n/messages_en_US] - neither plain properties nor XML
2021-05-03 12:03:56.890 DEBUG 1843 --- [  XNIO-1 task-9] o.o.webgoat.service.LabelDebugService    : Checking label debugging, it is false
2021-05-03 12:08:15.275 DEBUG 1843 --- [  XNIO-1 task-3] o.o.webgoat.service.LabelDebugService    : Checking label debugging, it is false
2021-05-03 12:08:22.771 DEBUG 1843 --- [  XNIO-1 task-5] o.o.webgoat.service.LabelDebugService    : Checking label debugging, it is false
2021-05-03 12:09:40.101 DEBUG 1843 --- [  XNIO-1 task-5] o.o.webgoat.service.LabelDebugService    : Checking label debugging, it is false

And also in the WebGoat website I can't access the the dashboard.
I get just this. Can anyone help?
![image](https://user-images.githubusercontent.com/74931435/116860011-933f1480-ac09-11eb-94de-c948dde3d523.png)

**Comment by nbaars** [comment]
@Cyrus254Kenya can you try to explicitly set the locale with:

`java -Duser.language=en-US` -jar ....`

**Comment by Cyrus254Kenya** [comment]
Hi.....still getting the same

**Comment by nbaars** [comment]
Ok, I will need to look at bit more closely, we've encountered this issue sometimes when the locale on the machine does not match with EN/US.

**Comment by Cyrus254Kenya** [comment]
So does this mean I cannot do anything on the dashboard?....
Because everytime I click anything it does not open.

**Comment by nbaars** [comment]
Indeed, both the lesson content the labels cannot be found, for example `sql.injection.title` refers to a property in the resource bundle. Did you try the Docker image or the standalone jar file?

**Comment by Cyrus254Kenya** [comment]
The standalone jar file.

**Comment by zubcevic** [comment]
#1297 I don't think this issue still occurs on the latest jar you can build from the develop branch or in the jar from the pull request where also a language switch is built in to override the browser locale for a user session.
